patronage/bubs-next
# feat: bumping husky to version 7

To test:

* pull
* `cd website`
* `yarn install`
* add junk to the top of a JS file
* go to commit it.

You should see the following, and you won't get to the commit stage.

```
yarn run v1.22.10
$ /Users/cameron/projects/bubs-next/website/node_modules/.bin/lint-staged
✔ Preparing...
⚠ Running tasks...
  ❯ Running tasks for *.{js,jsx,ts,tsx}
    ✖ eslint --fix [FAILED]
  ↓ No staged files match *.{css,scss} [SKIPPED]
↓ Skipped because of errors from tasks. [SKIPPED]
✔ Reverting to original state because of errors...
✔ Cleaning up...
```
